input.txt

```diff
@@ -0,0 +1,13 @@
+ANDREW PHILLIPS FELLOW Y
```

Comment:
**Code Quality: Nitpick**

You should try extending the list of people to see how well your program deals with large data sets.